docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -778,37 +770,35 @@ example:
 ----
 
 
-You might decide that you don't want to send the whole command-line argument to
-{es}. You can write a grok filter that matches the first part of the command
-and saves it to a new field. You can also remove the original field, `cmdline`,
-from the event. For example:
-
-// REVIEWERS: I took a stab at this, but I'm not very good at grok or
-// regex. In fact, I sort of hate it...a lot. Can someone review and suggest
-// something better based on the format you would expect in this field?
+Rather than sending the whole command-line argument to {es}, you might want to
+send just the command's path. One way to do that is by using a Grok filter.
```

Comment:
I don't like that this is wordier, but I want to make it clear that the grok filter is just one of the ways.